src/code/firebase/init.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Since we can't run a service worker from https://intel.ingress.com, we
+ * are creating an iframe that loads content hosted on server.wasabee.com.
```

Comment:
cdn2... not server...

Reply:
not .com too... I didn't write this comment ^^